modules/ob-release-notes-1-5-1.adoc

```diff
@@ -0,0 +1,24 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id="ob-release-notes-1-5-1_{context}"]
+= Release notes for {builds-shortname} 1.5.1
+
+{builds-shortname} 1.5 is now available on {ocp-product-title} 4.19.
+
+[id="new-features-1-5-1_{context}"]
+== New features
+
+The following sections highlight what is new in {builds-shortname} 1.5.1.
+
+* With this update, {builds-shortname} deployment is FIPS compliant.
+
+[id="fixed-issues-1-5-1_{context}"]
+== Fixed issues
+
+The following list highlights fixed issues in {builds-shortname} 1.5.1.
+
+* Before this update, adding volumes to `Build` and `Buildrun` custom resources failed because of the empty fields in Custom Resource Definition(CRD). With this update, the empty fields in CRD are removed.
+* Before this update, using `ExtKeyUsageAny` disabled policy validation in crypto/x509. This affected certificate chains that contain policy graphs. This issue is now fixed. link:https://www.cve.org/CVERecord?id=CVE-2025-22874[CVE-2025-22874] 
+* Before this update, the correct build container images were not pulled in a disconnected cluster. With this fix, the SHAs of the images are updated and synced between the operator bundle and the build strategies. This ensures that correct build container images are pulled in a disconnected cluster.
```

Comment:
Before this update, the correct build container images were not pulled in disconnected clusters because the image SHAs were not synchronized. With this update, the image SHAs are updated and synchronized between the operator bundle and the build strategies, ensuring that the correct build container images are pulled in a disconnected cluster.